lib/amplitude_web.dart

```diff
@@ -48,11 +48,27 @@ class AmplitudeFlutterPlugin {
         JSObject event = getEvent(call);
         amplitude.track(event);
       }
+      case "getUserId":
+      {
+        JSString? userId = amplitude.getUserId();
+        if (userId == null) {
+          return null;
+        }
+        return userId.toDart;
```

Comment:
nit: can we use null-aware operator here?
```suggestion
        return amplitude.getUserId()?.toDart;
```